ZAP2/ZAnimation.cpp

```diff
@@ -80,28 +80,30 @@ string ZAnimation::GetSourceOutputCode(string prefix)
 {
 	if (parent != nullptr)
 	{
-		string headerStr = StringHelper::Sprintf("\t { %i, %i }, %s_values, %s_indices, %i",
-			frameCount, 0, name.c_str(), name.c_str(), limit);
+		string headerStr = StringHelper::Sprintf("%i, %s_values, %s_indices, %i",
+			frameCount, name.c_str(), name.c_str(), limit);
 		parent->declarations[rawDataIndex] = new Declaration(DeclarationAlignment::None, 16, "AnimationHeader", StringHelper::Sprintf("%s", name.c_str()), false, headerStr);
 
 		string indicesStr = "";
 		string valuesStr = "\t";
+		int lineLength = 15;
```

Comment:
Minor thing but if these two lines aren't going to change, please make them constant.

Reply:
Done.